src/main/java/co/omise/models/schedules/Occurrence.java

```diff
@@ -21,11 +21,11 @@
 public class Occurrence extends Model {
     private String schedule;
     @JsonProperty("schedule_date")
-    private DateTime scheduleDate;
+    private LocalDate scheduleDate;
     @JsonProperty("retry_date")
-    private DateTime retryDate;
+    private LocalDate retryDate;
     @JsonProperty("processed_at")
-    private DateTime processedAt;
+    private ZonedDateTime processedAt;
     private OccurrenceStatus status;
```

Comment:
How did you decide that the other two are `LocalDate` and this one is `ZonedDateTime`?

Reply:
Depend on response `scheduleDate` and `retryDate` return response only date e.g. `2018-01-01` but `processedAt` return full date & timezone `2018-01-01T12:00:00Z`

Reply:
Okk got it 👌